test/dk/aau/cs/ds306e18/tournament/rlbot/configuration/MatchConfigTest.java

```diff
@@ -0,0 +1,68 @@
+package dk.aau.cs.ds306e18.tournament.rlbot.configuration;
+
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import static dk.aau.cs.ds306e18.tournament.rlbot.configuration.MatchConfigOptions.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+public class MatchConfigTest {
+
+    private static final String MATCH_CONFIG_FILE = "test/rlbot.cfg";
+    private static final String MATCH_CONFIG_FILE_OUT = "test/_out_rlbot.cfg";
+
+    @Test
+    public void read01() throws IOException {
+        MatchConfig config = new MatchConfig(new File(MATCH_CONFIG_FILE));
+
+        // Match settings
+        assertEquals(GameMap.MANNFIELD, config.getGameMap());
+        assertEquals(GameMode.SOCCER, config.getGameMode());
+        assertFalse(config.isSkipReplays());
+        assertFalse(config.isInstantStart());
+
+        // Mutators
+        assertEquals(MatchLength.UNLIMITED, config.getMatchLength());
+        assertEquals(MaxScore.UNLIMITED, config.getMaxScore());
+        assertEquals(Overtime.UNLIMITED, config.getOvertime());
+        assertEquals(BallMaxSpeed.DEFAULT, config.getBallMaxSpeed());
+        assertEquals(GameSpeed.DEFAULT, config.getGameSpeed());
+        assertEquals(BallType.DEFAULT, config.getBallType());
+        assertEquals(BallWeight.DEFAULT, config.getBallWeight());
+        assertEquals(BallSize.DEFAULT, config.getBallSize());
+        assertEquals(BallBounciness.DEFAULT, config.getBallBounciness());
+        assertEquals(BoostAmount.DEFAULT, config.getBoostAmount());
+        assertEquals(BoostStrength.TIMES_ONE, config.getBoostStrength());
+        assertEquals(RumblePowers.NONE, config.getRumblePowers());
+        assertEquals(Gravity.DEFAULT, config.getGravity());
+        assertEquals(Demolish.DEFAULT, config.getDemolish());
+        assertEquals(RespawnTime.THREE_SECONDS, config.getRespawnTime());
+
+        // Participants
+        // Note that we can't load the BotConfigs as their path is not and can't be absolute. But we can still test
```

Comment:
> ... their path is not ...

Not what?

Reply:
Hm, not sure actually. Something with resolving paths, absolute paths, and unnecessity. We don't need to test if the paths are correct or if BotConfigs work as there is already a test for that.

Reply:
Hm. Okay, if you can't recall what it should be, then lets remove that part so it doesn't cause further confusion.

Reply:
Resolved in 46b17a61d322984d5e3cbe61f85652b151d414c7